File: apps/institution-api/Models/VwCurrentInstitutionLocation.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionLocation
    {
        public int InstitutionId { get; set; }
        public int InstitutionLocationId { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string LocationName { get; set; }
        public int? LocationNumber { get; set; }
        public string LocationStatus { get; set; }
        public bool? IsOffSite { get; set; }
        public Guid? LocationGuid { get; set; }
    }
}

File: libs/common/models/vw-current-institution-location.ts



export class VwCurrentInstitutionLocation {
    public institutionId: number = 0;
    public institutionLocationId: number = 0;
    public institutionName: string = '';
    public institutionCode: string = '';
    public locationName: string = '';
    public locationNumber: number;
    public locationStatus: string = '';
    public isOffSite: boolean;
    public locationGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionLocation>) {
        
        Object.assign(this, init);
    }
}

